fdbserver/workloads/BackupToDBCorrectness.actor.cpp

```diff
@@ -660,6 +683,11 @@ struct BackupToDBCorrectnessWorkload : TestWorkload {
 
 				wait(success(restoreTool.waitBackup(cx, self->restoreTag)));
 				wait(restoreTool.unlockBackup(cx, self->restoreTag));
+
```

Comment:
The cycle test does not change the number of KV pairs in a key range, it just updates the value of existing keys, so this validation needs to compare the actual KV pairs